home.html

```diff
@@ -57,7 +57,7 @@ <h3 class="panel-title">Bookmarklet</h3>
   <div class="panel-body">
     <p>Drag this button to your bookmarks bar:
     <a class="btn btn-primary btn-xs" role="button"
-       href="javascript:(function(){m=location.href.match(/(way|node|relation).(\d+)/);window.open('https://pewu.github.io/osm-history/#/'+(m?m[1]+'/'+m[2]:''));})()">
+       href="javascript:(function(){m=window.getSelection().toString().toLowerCase();if(m==(""||null||false)){m=location.href;}m=m.match(/(way|node|relation)\D+(\d+)/);window.open('https://pewu.github.io/osm-history/#/'+(m?m[1]+'/'+m[2]:''));})()">
```

Comment:
You can make it a bit more concise:
m=(window.getSelection().toString().toLowerCase()||location.href).match(...
Instead of comparing to (""||null||false) you can rely on the fact that "" is falsy. Additionally, the || operator is quite handy here that takes the second value if the first one is falsy.

Reply:
Cool. Thank you